plugins/apache-airflow/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@backstage/plugin-apache-airflow",
+  "version": "0.0.0",
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "license": "Apache-2.0",
+  "publishConfig": {
+    "access": "public",
+    "main": "dist/index.esm.js",
+    "types": "dist/index.d.ts"
+  },
+  "scripts": {
+    "build": "backstage-cli plugin:build",
+    "start": "backstage-cli plugin:serve",
+    "lint": "backstage-cli lint",
+    "test": "backstage-cli test",
+    "diff": "backstage-cli plugin:diff",
+    "prepack": "backstage-cli prepack",
+    "postpack": "backstage-cli postpack",
+    "clean": "backstage-cli clean"
+  },
+  "dependencies": {
+    "@backstage/core-components": "^0.7.4",
+    "@backstage/core-plugin-api": "^0.2.0",
+    "@backstage/theme": "^0.2.13",
+    "@material-ui/core": "^4.12.2",
+    "@material-ui/icons": "^4.9.1",
+    "@material-ui/lab": "4.0.0-alpha.57",
+    "cross-fetch": "^3.0.6",
+    "qs": "^6.10.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "react-use": "^17.2.4"
+  },
+  "devDependencies": {
+    "@backstage/cli": "^0.9.0",
```

Comment:
happen to recall the latest cli version is 0.100; might want to bump here, and/or just check the latest list of deps is used? (though, and maintainers can correct me, i'd guess once merged in, the next round of bumps by the automation will take care of this?)